### PR TITLE
Add parboiled2 to community build

### PR DESCRIPTION
**Pull request by jrudolph**
It works at the commit where I branched off but fails because of the problem discussed in https://github.com/lampepfl/dotty/issues/14903#issuecomment-1096632438

### CODE REVIEW

**Comment by smarter** [comment]
Could you try rebasing to see if this is now fixed?

File: .gitmodules [hunk 1]
```diff
@@ -219,3 +219,6 @@
 [submodule "community-build/community-projects/http4s"]
 	path = community-build/community-projects/http4s
 	url = https://github.com/dotty-staging/http4s.git
+[submodule "community-build/community-projects/parboiled2"]
+	path = community-build/community-projects/parboiled2
+	url = https://github.com/sirthias/parboiled2.git
```
Comment (griggt) [inline .gitmodules:224]:
```suggestion
	url = https://github.com/dotty-staging/parboiled2.git
```

File: community-build/src/scala/dotty/communitybuild/projects.scala [hunk 2]
```diff
@@ -761,6 +761,14 @@ object projects:
     dependencies = List(cats, catsEffect3, fs2, disciplineMunit, scalacheckEffect)
   )
 
+  lazy val parboiled2 = SbtCommunityProject(
+    project = "parboiled2",
+    sbtTestCommand = "parboiledCoreJVM/test; parboiledJVM/test",
+    sbtPublishCommand = "publishLocal",
+    scalacOptions = SbtCommunityProject.scalacOptions.filter(_ != "-Xcheck-macros"),
+    dependencies = List(utest, scalatest)
```
Comment (griggt) [inline community-build/src/scala/dotty/communitybuild/projects.scala:769]:
```suggestion
    dependencies = List(utest, scalacheck)
```

**Review by griggt — APPROVED** [review]

File: community-build/src/scala/dotty/communitybuild/projects.scala (hunk 2 above)
Comment (jrudolph) [reply community-build/src/scala/dotty/communitybuild/projects.scala]: Oops, you are right. Thanks for fixing!

**Comment by jrudolph** [comment]
Great, thanks!

**Pull request merged**